### PR TITLE
feat(contract_manager): add a script to deploy evm contract

### PR DESCRIPTION
**Pull request by ali-bahjati**

### CODE REVIEW

File: contract_manager/scripts/deploy_evm_contract.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "deploy-args": {
+      type: "array",
+      desc: "Arguments to pass to the contract constructor",
+    },
+  });
+
+async function main() {
+  const argv = await parser.argv;
+
+  const chain = DefaultStore.chains[argv.chain] as EvmChain;
```
Comment (m30m) [inline contract_manager/scripts/deploy_evm_contract.ts:38]: maybe a little bit of error handling here? check if the chain does not exist or is not evm perhaps?

File: contract_manager/scripts/deploy_evm_contract.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      desc: "Private key to use for the deployment",
+    },
+    chain: {
+      type: "string",
+      demandOption: true,
+      desc: "Chain to upload the contract on. Can be one of the evm chains available in the store",
+    },
+    "deploy-args": {
+      type: "array",
+      desc: "Arguments to pass to the contract constructor",
```
Comment (m30m) [inline contract_manager/scripts/deploy_evm_contract.ts:31]: mention the format, it's hex with leading `0x` or not ...

**Review by m30m — APPROVED** [review]
left minor comments

File: contract_manager/scripts/deploy_evm_contract.ts (hunk 2 above)
Comment (ali-bahjati) [reply contract_manager/scripts/deploy_evm_contract.ts:31]: I really don't know what the format is. In the code it takes `any[]`. I'll add a FIXME.

Comment (ali-bahjati) [reply contract_manager/scripts/deploy_evm_contract.ts]: well i think i found it

File: contract_manager/scripts/deploy_evm_contract.ts (hunk 1 above)
Comment (ali-bahjati) [reply contract_manager/scripts/deploy_evm_contract.ts]: done

**Pull request merged**